tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,86 @@
+        for holiday_date, holiday_name in fixed_holidays.items():
+            self.assertIn(holiday_date, self.holidays_2024)
+            self.assertEqual(self.holidays_2024[holiday_date], holiday_name)
+
+    
+    def test_historical_date_changes(self):
+        """Test holidays that changed dates over time."""
+        self.assertIn(date(2017, 10, 27), self.holidays_2017)
+        self.assertEqual(self.holidays_2017[date(2017, 10, 27)], "Independence Day")
+
+        self.assertIn(date(2024, 9, 27), self.holidays_2024)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)